freqtrade/edge/__init__.py

```diff
@@ -52,8 +53,17 @@ def __init__(self, config: Dict[str, Any], exchange, strategy) -> None:
 
         self.edge_config = self.config.get('edge', {})
         self._cached_pairs: Dict[str, Any] = {}  # Keeps a list of pairs
+        self._final: list = []
 
-        self._total_capital: float = self.config['stake_amount']
+        # checking max_open_trades. it should be -1 as with Edge
+        # the number of trades is determined by position size
+        if self.config['max_open_trades'] != -1:
+            logger.critical('max_open_trades should be -1 in config !')
+
+        if self.config['stake_amount'] != constants.UNLIMITED_STAKE_AMOUNT:
+            raise OperationalException('Edge works only with unlimited stake amount')
+
+        self._capital_percentage: float = self.edge_config.get('capital_available_percentage')
```

Comment:
i think edge will fail miserably if i don't have `"capital_available_percentage"` in my config but edge enabled (current configuration, basically) - because capital_percent will be None in that case, but no error is shown.

adding it to "required" as mentioned above should mitigate that though...